### ISSUE TITLE
Feature: Store output files in sub folder

### ISSUE THREAD

**Issue by brianmangan** [nyavramov/WEBMARIZER#2]
like the topic says,

Add checkbox option to put output files in sub folder. 

/WEBMARIZER/ would be appropriate.

**Comment by nyavramov** [comment]
Sure thing, man. Adding this to my to-do list!

**Comment by brianmangan** [comment]
see pull request with feature added. sorry about the reformat.. stupid pycharm thinks it knows better.

**Comment by nyavramov** [comment]
I will take a look at this very soon. Just uploaded my refactored code at the exact same time you sent the pull-request - crazy timing :)!

**Comment by brianmangan** [comment]
I see... well that blows everything out of the water I was doing..

**Comment by brianmangan** [comment]
here is what I added, but since you refactor.. you can copy paste... 

New Global Variables
```
output_to_subfolder = True
output_subfolder = 'output'
```

New Function:
```
def generateOuput_filename(fileName, extension, numFiles):
    if output_to_subfolder:
        if not os.path.isdir(output_subfolder):
            os.makedirs(output_subfolder)
        return os.path.join(output_subfolder, os.path.splitext(fileName)[0] + '_' + str(numFiles) + '.' + extension)
```

Changing the output file name value to use new fuction
```
fileName_webm = generateOuput_filename(fileName=fileName, extension='webm', numFiles=numFiles)
```

**Comment by brianmangan** [comment]
See #4 for feature being added. 

needs UI changes to make it official, but this works for now.. lemme know what you think.

**Comment by brianmangan** [comment]
resolved by #4 

Closing
